packages/ui/src/ui/pages/query-tracker/module/query/actions.ts

```diff
@@ -249,7 +249,16 @@ export function runQuery(
     return async (dispatch, getState) => {
         const state = getState();
         const query = getQueryDraft(state);
-        const {query_id} = await wrapApiPromiseByToaster(dispatch(startQuery(query, options)), {
+
+        //new queries should not inherit SHARED_QUERY_ACO
+        const newQuery = {
+            ...query,
+            access_control_objects: query.access_control_objects?.filter(
```

Comment:
It looks like the PR should be modified after merging of https://github.com/ytsaurus/ytsaurus-ui/pull/711.
access_control_objects should be modified only when `isMultipleAco == true`

Reply:
yes, this branch is best merge after https://github.com/ytsaurus/ytsaurus-ui/pull/711